test-site/public/iframe_test.html

```diff
@@ -1 +1 @@
-<html><head><meta name="viewport" content="initial-scale=1,maximum-scale=1,minimum-scale=1,user-scalable=no,width=device-width"></head><body><div id="answers-container"></div><script src="iframe.js"></script></body></html>
\ No newline at end of file
+<html><head><meta name="viewport" content="initial-scale=1,maximum-scale=1,minimum-scale=1,user-scalable=no,width=device-width"></head><body><div id="answers-container" data-hhconfig-test="test-data"></div><script src="iframe.js"></script></body></html>
```

Comment:
are we committing this file just for easier manual testing in the future?

Reply:
Ya, I feel that it's helpful to have an example of using the data attribute